pydicom/values.py

```diff
@@ -192,26 +262,32 @@ def convert_PN(byte_string, is_little_endian, struct_format=None, encoding=None)
 
     # We would like to return string literals
     if not in_py2:
-        valtype = lambda x: PersonName(x, *args).decode()
+        valtype = lambda x: PersonName(x, *args).decode()  # nopep8
     else:
-        valtype = lambda x: PersonName(x, *args)
+        valtype = lambda x: PersonName(x, *args)  # nopep8
 
```

Comment:
Could you give an example of how to implement it? I had done a function like:

```
def calc_valtype(x):
    if not in_py2:
        return PersonName(x, *args).decode()
    else:
        return PersonName(x, *args)
```

and then used it like:

```
   if len(splitup) == 1:
        return calc_valtype(splitup[0])
    else:
        return MultiValue(calc_valtype, splitup)
```
and the tests failed gloriously, I think because the MultiValue needs something special handed to it. I decided to be conservative and choose the fugly approach rather than breaking the code. If someone who is familiar with how it should work has a suggestion to try I can throw it in! We can always go back if things get awry.